src/chttpd_db.erl

```diff
@@ -74,6 +86,8 @@ handle_changes_req1(#httpd{}=Req, Db) ->
     ChangesArgs = Args0#changes_args{
         filter_fun = couch_changes:configure_filter(Raw, Style, Req, Db)
     },
+    % Default to ~filling the payload of a standard Ethernet frame
+    Max = config:get_integer("httpd", "chunked_response_buffer", 1490),
```

Comment:
I believe, this option we should advice to tweak if user iface is configured for some other MTU, right? Also, standard Ethernet frame is 1500, why 1490 had chosen here?